airbyte-integrations/connectors/source-airtable/acceptance-test-config.yml

```diff
@@ -11,18 +12,27 @@ acceptance_tests:
         status: "succeed"
       - config_path: "integration_tests/invalid_config.json"
         status: "failed"
+      - config_path: "secrets/config_oauth.json"
+        status: "succeed"
+      - config_path: "integration_tests/invalid_config_oauth.json"
+        status: "failed" 
   discovery:
     tests:
       - config_path: "secrets/config.json"
       # bypassed this check, because discovery mechanism was changed
         backward_compatibility_tests_config:
-          disable_for_version: "0.1.3"
+          disable_for_version: "1.0.1"
```

Comment:
Is this the same bypass reason as for 0.1.3?

Reply:
This part could be removed for this particular version release, there are no changes with the previous version, so it should be backward capable with the previous version.